docs/security.md

```diff
@@ -20,6 +20,18 @@ To change the SSH public key on an existing cluster:
 * `kops update cluster --yes` to reconfigure the auto-scaling groups
 * `kops rolling-update cluster --name <clustername> --yes` to immediately roll all the machines so they have the new key (optional)
 
+### EC2 Instance Connect
+
+Kops clusters on AWS can configure make use of [EC2 Instance Connect](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Connect-using-EC2-Instance-Connect.html).
+
+To enable this, add the following to the Instance Group spec.
+
+```
+spec:
+    ec2InstanceConnect: true
```

Comment:
I don't think we're doing a wonderful job of this at present, but I think it would be sensible to adopt a standard for cloud provider specific API fields. I'm not sure we have a great way to do this right now. 

@johngmyers @justinsb @rifelpet @hakman (a random sample of kops folks that popped up in my autocomplete :) -
Do we have a way to namespace fields like this on a node or cluster spec. Would it be at all useful to do something like `nodepool.spec.aws.aws_specific_field` or should config like this live at root?

If we had a way to namespace API fields by provider, I would prefer this field to be named something like `enableInstanceConnect` to make it clear what the `true/false` value does.

Reply:
I realize that we have a lot of legacy properties that strictly may need to be namespaced, but I am proposing no changes to any existing properties unless others have a strong desire to port them. I was just thinking about new, clearly cloud provider specific properties, like this. 

Alternatively, this feature sounds roughly analogous to GCP/os-login. We could theoretically do a field that was like `enableCloudProviderLogin` and support these cases and potentially others under a single flag. My money is on having provider specific flags that clearly enable or disable a cloud specific feature, for simplicity, to make it obvious that something is definitely supported or unsupported, etc.